pfs/lineLists/HgCd.txt

```diff
@@ -15,18 +15,321 @@
+ 734.30000            1.00        HgI       0
+ 737.35000           10.00        HgI       0
+ 737.65000            4.00        HgI       0
+ 738.95000            1.00        HgI       0
+ 739.58000            1.00        HgI       0
+ 740.53000            1.00        HgI       0
+ 741.42000            4.00        HgI       0
+ 745.60000            8.00        HgI       0
+ 745.88000            3.00        HgI       0
+ 746.34000            2.00        HgI       0
+ 747.50000            1.00        HgI       0
+ 749.66000            1.00        HgI       0
+ 750.25000            1.00        HgI       0
+ 752.22000            9.00        HgI       0
+ 753.37000            1.00        HgI       0
+ 753.60000            0.00        HgI       0
+ 754.50000            4.00        HgI       0
+ 755.42000            7.00        HgI       0
+ 756.29000            1.00        HgI       0
+ 757.33000            1.00        HgI       0
+ 758.30000            1.00        HgI       0
+ 760.42000            7.00        HgI       0
+ 760.80000            4.00        HgI       0
+ 761.48000            1.00        HgI       0
+ 761.76000            0.00        HgI       0
+ 762.09000            1.00        HgI       0
+ 762.29000            0.00        HgI       0
+ 763.38000            1.00        HgI       0
+ 763.73000            0.00        HgI       0
+ 764.85000            3.00        HgI       0
+ 765.58000            1.00        HgI       0
+ 767.61000            5.00        HgI       0
+ 767.94000            3.00        HgI       0
+ 768.73000            1.00        HgI       0
+ 769.51000            1.00        HgI       0
+ 770.62000            1.00        HgI       0
+ 770.87000            1.00        HgI       0
+ 773.09520           30.00        HgI       0
+ 773.47000            5.00        HgI       0
+ 773.68000            0.00        HgI       0
+ 779.73000            1.00        HgI       0
+ 782.82000            4.00        HgI       0
+ 790.13000            0.00        HgI       0
+ 793.20000            3.00        HgI       0
+ 793.91000            1.00        HgI       0
+ 795.34000            0.00        HgI       0
+ 797.08000            0.00        HgI       0
+ 797.55000            1.00        HgI       0
+ 798.13000            7.00        HgI       0
+ 799.32000            0.00        HgI       0
+ 800.08000            0.00        HgI       0
+ 800.93000            1.00        HgI       0
+ 801.55000            0.00        HgI       0
+ 802.01000            1.00        HgI       0
+ 802.45000            6.00        HgI       0
+ 803.41000             nan        HgI       0
+ 805.22000            1.00        HgI       0
+ 805.63000            2.00        HgI       0
+ 806.06000            1.00        HgI       0
+ 807.29000            5.00        HgI       0
+ 808.03000             nan        HgI       0
+ 808.26000            0.00        HgI       0
+ 809.26000             nan        HgI       0
```

Comment:
@PaulPrice are the `nan`s listed in the intensity column going to cause any problems?

Reply:
I believe they've been in there previously without any problem. But please do test this linelist on real data!